parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -309,7 +309,7 @@ disableSystemdResolved() {
     ls -ltr /etc/resolv.conf
     cat /etc/resolv.conf
     UBUNTU_RELEASE=$(lsb_release -r -s 2>/dev/null || echo "")
-    if [ "${UBUNTU_RELEASE}" = "18.04" ] || [ "${UBUNTU_RELEASE}" = "20.04" ] || [ "${UBUNTU_RELEASE}" = "22.04" ] || [ "${UBUNTU_RELEASE}" = "24.04" ]; then
+    if [ "${UBUNTU_RELEASE}" = "20.04" ] || [ "${UBUNTU_RELEASE}" = "22.04" ] || [ "${UBUNTU_RELEASE}" = "24.04" ]; then
```

Comment:
so this is where we need to take a s hard stance on whether we're just gonna break customers who are currently using 1804 - if you modify CSE/provisioning scripts to no longer support 1804 whatsoever, then any customer running an 1804 nodepool that performs a reimaging operation (such as updating their kubelet identity / service principal credential for example) will get a new VMSS model still referencing their original node image (since we froze them to 202506.03.0) but will get the latest CSE that no longer supports that node image

this is the same reason why we still have this docker logic hanging around fyi even though that's been out of support for almost 3 years at this point

Reply:
while we're at it we should cleanup all docker support then, that's a separate PR though